client/dataEntry.js

```diff
@@ -406,25 +397,17 @@ Template.roundDataEntry.events({
     // We save when the user leaves the currently focused jChester.
     let $jChester = $(e.currentTarget);
 
-    let focusNewJChester = function() {
-      Tracker.afterFlush(function() {
-        let $jChesterNew = $(document.activeElement).closest('.jChester');
-        if($jChesterNew[0] !== $jChester[0]) {
-          $jChesterNew.focus();
-          let $warningIconNew = $jChesterNew.parents("tr").find('i[data-toggle="popover"]');
-          $('[data-toggle="popover"]').not($warningIconNew).popover('hide');
-        }
-      });
-    };
+    Tracker.afterFlush(function() {
+      let $jChesterNew = $(document.activeElement).closest('.jChester');
+      if($jChesterNew[0] !== $jChester[0]) {
+        $jChesterNew.focus();
+        let $warningIconNew = $jChesterNew.parents("tr").find('i[data-toggle="popover"]');
+        $('[data-toggle="popover"]').not($warningIconNew).popover('hide');
+      }
+    });
 
-    // Focus the new jChester. If we do this immediately, document.activeElement is still
-    // the body, which isn't useful.
-    focusNewJChester();
+    updateWarningContents();
```

Comment:
Hmm... does it work to call `updateWarningContents` here? Doesn't `updateWarningContents` use the solves in the database, which we aren't updating anymore until the user clicks save-button?